File: java-play-framework/app/apimodels/ResponseTransportationMode.java
package apimodels;

import com.fasterxml.jackson.annotation.*;
import java.util.Set;
import javax.validation.*;
import com.fasterxml.jackson.annotation.JsonCreator;

/**
 * Gets or Sets ResponseTransportationMode
 */
public enum ResponseTransportationMode {
  
  CAR("car"),
  
  PARKING("parking"),
  
  BOARDING("boarding"),
  
  WALK("walk"),
  
  BIKE("bike"),
  
  TRAIN("train"),
  
  RAIL_NATIONAL("rail_national"),
  
  RAIL_OVERGROUND("rail_overground"),
  
  RAIL_UNDERGROUND("rail_underground"),
  
  RAIL_DLR("rail_dlr"),
  
  BUS("bus"),
  
  CABLE_CAR("cable_car"),
  
  PLANE("plane"),
  
  FERRY("ferry"),
  
  COACH("coach");

  private final String value;

  ResponseTransportationMode(String value) {
    this.value = value;
  }

  @Override
  @JsonValue
  public String toString() {
    return String.valueOf(value);
  }

  @JsonCreator
  public static ResponseTransportationMode fromValue(String value) {
    for (ResponseTransportationMode b : ResponseTransportationMode.values()) {
      if (b.value.equals(value)) {
        return b;
      }
    }
    throw new IllegalArgumentException("Unexpected value '" + value + "'");
  }
}

File: fsharp-functions/OpenAPI/src/model/ResponseTransportationMode.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json

module ResponseTransportationMode = 

  //#region ResponseTransportationMode

  let CarEnum = "car"
  let ParkingEnum = "parking"
  let BoardingEnum = "boarding"
  let WalkEnum = "walk"
  let BikeEnum = "bike"
  let TrainEnum = "train"
  let RailNationalEnum = "rail_national"
  let RailOvergroundEnum = "rail_overground"
  let RailUndergroundEnum = "rail_underground"
  let RailDlrEnum = "rail_dlr"
  let BusEnum = "bus"
  let CableCarEnum = "cable_car"
  let PlaneEnum = "plane"
  let FerryEnum = "ferry"
  let CoachEnum = "coach"
  type ResponseTransportationMode = string
  
  //#endregion
  